src/l3/ip.rs

```diff
@@ -171,3 +154,37 @@ impl Ipv6Hdr {
         bitfield_unit
     }
 }
+
+/// Protocol which is encapsulated in the IPv4 packet.
+/// <https://en.wikipedia.org/wiki/List_of_IP_protocol_numbers>
```

Comment:
I would link to IANA as the canonical source: https://www.iana.org/assignments/protocol-numbers/protocol-numbers.xhtml